models/repo.go

```diff
@@ -1715,12 +1716,12 @@ func DeleteRepository(doer *User, uid, repoID int64) error {
 
 	// Remove issue attachment files.
 	for i := range attachmentPaths {
-		removeAllWithNotice(x, "Delete issue attachment", attachmentPaths[i])
+		RemoveStorageWithNotice(storage.Attachments, "Delete issue attachment", attachmentPaths[i])
```

Comment:
This will potentially be very slow (e.g. with a migrated repository). Maybe it should be handled in background or even better on a queue.

Reply:
Right, but I will leave it as another PR to not make this PR too fat.